src/test/java/com/game/Battleship/repository/BoardRepositoryTests.java

```diff
@@ -0,0 +1,29 @@
+package com.game.Battleship.repository;
+
+import com.game.Battleship.models.Board;
+import com.game.Battleship.models.Ship;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class BoardRepositoryTests {
+
+    @Test
+    public void updateBoardWithShips(){
+        Board board = new Board();
+        Ship ship = new Ship("Destroyer", 2 );
+        BoardRepository boardRepository = new BoardRepository();
+        boardRepository.updateBoardWithShips(board, ship, 0, 1);
+        Assertions.assertEquals(board.getBoardField().get(1).getValue(), "Ship");
+
+    }
+
+    @Test
+    public void updateBoardWithStrike(){
+        Board board = new Board();
+        BoardRepository boardRepository = new BoardRepository();
+        boardRepository.updateBoardWithStrike(board, 0, 1 );
+        Assertions.assertEquals(board.getBoardField().get(1).getValue(), "Missed");
+
+    }
+
```

Comment:
It looks like you tested the missed case in here but what about the strike case?

Reply:
Added